config/secrets.yml

```diff
@@ -12,7 +12,10 @@
 
 development:
   secret_key_base: eef0113a80061211ba4f7388821508811be2717f5734e6160050f5425b3e526f64444c154d7ceee1bebceeb850609baed3684222f435cf0641097afb5d430478
-
+sit:
```

Comment:
this may have some useful info also https://stackoverflow.com/questions/25426940/what-is-the-use-of-secret-key-base-in-rails-4

Reply:
Agree, the codebase came with these in git but I think we should create a ticket to put these in env variables instead, this will take some coordination with cicd as well.

Reply:
ok, but now more are being added with the new environments. I guess this is already in dev, this merge doesn't change much. But I would like to see us have a high priority ticket to resolve the problem.

Reply:
Alright, I created a ticket CMRARC-422, scheduled for this sprint.